search-app/docs/api.md

```diff
@@ -235,6 +235,7 @@ These are query parameters that control what extra data is included with collect
   * `include_has_granules` - If this parameter is set to "true" this will include a flag indicating true or false if the collection has any granules at all. Supported in all response formats except opendata. To limit search results to collections with or without granules, see the [`has_granules`](#c-has-granules) parameter.
   * `include_granule_counts` - If this parameter is set to "true" this will include a count of the granules in each collection that would match the spatial and temporal conditions from the collection query. Supported in all response formats except opendata.
   * `include_facets` There are 3 values allowed: "true", "false", or "v2". If this parameter is set to "true" or "v2" facets will be included in the collection results (not applicable to opendata results). Facets are described in detail below.
+  * `facets_size` is used to customize the max number of values displayed in each facet. Values allowed: positive integer. 
```

Comment:
This is a interim fix, we don't have to update the doc. We should have the individual facet field in the description of this new search param when we do update the doc.